rpcs3/Emu/Cell/lv2/sys_ppu_thread.cpp

```diff
@@ -1,4 +1,4 @@
-﻿#include "stdafx.h"
+#include "stdafx.h"
```

Comment:
Bogus encoding change?

Reply:
Actually, it's removing the encoding marker.